atest/acceptance/keywords/click_element_modifier.robot

```diff
@@ -42,6 +42,13 @@ Click Element Wrong Modifier
     ...    ValueError: 'FOOBAR' modifier does not match to Selenium Keys
     ...    Click Element    Button    Foobar
 
+Click Element Action Chain and modifier
+    [Documentation]   Clicks an element with both a valid modifier and action_chain argument. Modifier should should take precedent
```

Comment:
Please remove the documentation saying what the test does, it doesn't bring value and only adds maintenance burden. If you want, test name could say: `In Click Element Modified Argument Must 
Overrule Action Chains argument`